crates/uv-workspace/src/pyproject.rs

```diff
@@ -353,6 +353,17 @@ pub struct ToolUv {
     )]
     pub default_groups: Option<DefaultGroups>,
 
+    /// Additional settings for `dependency-groups`
+    #[option(
+        default = "[]",
+        value_type = "dict",
+        example = r#"
+            [tool.uv.dependency-groups]
+            my-group = {requires-python = ">=3.12"}
+        "#
+    )]
+    pub dependency_groups: Option<ToolUvDependencyGroups>,
```

Comment:
We may need to document that you can't declare group members here and that those need to go in `[dependency-groups]`?

Reply:
Hmm interesting yeah.